### PR TITLE
README.md add Style bach

### PR DESCRIPTION
**Pull request by Frexis**
Add style bach code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +35,7 @@ Here is a sample session with the program:
 179022.html
 % ./categories --render 6666666666
 No category with ID: 6666666666
+```
 
 Example HTML Output:
```
Comment (sumelio) [inline README.md:41]: Please add style here

**Comment by Frexis** [comment]
Ready

**Review by Frexis — COMMENTED** [review]
OK

**Review by Frexis — COMMENTED** [review]
oK TWO

**Comment by sumelio** [comment]
Ready final

**Pull request merged**